commands/cloudapi-v5/backupunit.go

```diff
@@ -310,6 +327,47 @@ func getBackupUnitInfo(c *core.CommandConfig) *resources.BackupUnitProperties {
 	return &properties
 }
 
+func DeleteAllBackupUnits(c *core.CommandConfig) error {
+	_ = c.Printer.Print("Backup Unitts to be deleted:")
+	backupUnits, resp, err := c.CloudApiV5Services.BackupUnit().List()
+	if err != nil {
+		return err
+	}
+	if backupUnitsItems, ok := backupUnits.GetItemsOk(); ok && backupUnitsItems != nil {
+		for _, backupUnit := range *backupUnitsItems {
+			if id, ok := backupUnit.GetIdOk(); ok && id != nil {
+				_ = c.Printer.Print("BackupUnit Id: " + *id)
+			}
+			if properties, ok := backupUnit.GetPropertiesOk(); ok && properties != nil {
+				if name, ok := properties.GetNameOk(); ok && name != nil {
+					_ = c.Printer.Print(" BackupUnit Name: " + *name)
+				}
+			}
+		}
+
+		if err := utils.AskForConfirm(c.Stdin, c.Printer, "delete all the Backup Units"); err != nil {
+			return err
+		}
+		c.Printer.Verbose("Deleting all the BackupUnits...")
+		for _, backupUnit := range *backupUnitsItems {
+			if id, ok := backupUnit.GetIdOk(); ok && id != nil {
+				c.Printer.Verbose("Deleting Backup unit with id: %v...", *id)
+				resp, err = c.CloudApiV5Services.BackupUnit().Delete(*id)
+				if resp != nil {
+					c.Printer.Verbose(config.RequestTimeMessage, resp.RequestTime)
```

Comment:
In the `delete-all` functionality, the idea is that multiple requests are being sent in the same command. Printing only the request time doesn't help that much the user to know where or what to debug. In the simple functionality, this is not  much a big of a problem, because at the end of the command, the request id is still printed. (with the getPrint() function on return). 

I suggest adding also a log for printing the Request Id (right before the log with the request time). It can be used maybe `GetId()` function from the printer or something similar.

This applies to all the places in this PR with the delete all functionality.

Reply:
For delete all, first you cans see the resources ids and names that will be deleted, and when is performing de delete operation, the id of the resource is shown again, so you can correlate between ids to see what resources were actually deleted.
I will add de request id too for easy debug purose